### PR TITLE
Add example of emojiID to MessageReactionAdd

### PR DESCRIPTION
**Pull request by ceriath**
Since i ran into the same issue today, the documentation for MessageReactionAdd should be improved, thanks to #750

### CODE REVIEW

**Comment by CarsonHoffman** [comment]
I definitely think that the documentation could be improved here, and though an example is surely helpful, I think actually defining the format would be helpful as well—it might not be immediately clear that `hello:1234567654321` refers to `name:id`.

**Comment by adityaxdiwakar** [comment]
I might be wrong here but doesn't the format also require a colon, such as `:name:id`?
image

**Comment by CarsonHoffman** [comment]
The leading colon is necessary when specifying an emoji in the content of a message; it is not necessary when specifying an emoji in an API context. (I believe it won't fail as it will be considered part of the name, as the API seems to parse `name:id` from right-to-left starting from the ID and considering everything before the right-most colon as part of the name)

File: restapi.go [hunk 1]
```diff
@@ -2100,7 +2100,7 @@ func (s *Session) WebhookExecute(webhookID, token string, wait bool, data *Webho
 // MessageReactionAdd creates an emoji reaction to a message.
 // channelID : The channel ID.
 // messageID : The message ID.
-// emojiID   : Either the unicode emoji for the reaction, or a guild emoji identifier.
+// emojiID   : Either the unicode emoji for the reaction, or a guild emoji identifier (e.g. "hello:1234567654321").
```
Comment (FedorLap2006) [inline restapi.go:2103]: As Carson already pointed out, I think adding something like
```suggestion
// emojiID   : Either the unicode emoji for the reaction, or a guild emoji identifier in name:id format (e.g. "hello:1234567654321")
```
would be really nice

**Comment by FedorLap2006** [comment]
Sorry response took a long time.

**Comment by ceriath** [comment]
I added the suggestion

**Comment by FedorLap2006** [comment]
Thanks for your contribution!

**Pull request merged**